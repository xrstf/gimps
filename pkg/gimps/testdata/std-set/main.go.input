package main

import (
	"fmt"
	"go.xrstf.de/gimps/pkg/gimps"
	"log"
)

func main() {
	fmt.Println(gimps.Config{})
	log.Println("foo")
}
