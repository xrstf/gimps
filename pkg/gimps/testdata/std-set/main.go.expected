package main

import (
	"fmt"
	"log"

	"go.xrstf.de/gimps/pkg/gimps"
)

func main() {
	fmt.Println(gimps.Config{})
	log.Println("foo")
}
